modules/3d/src/rgbd/color_tsdf_functions.hpp

```diff
@@ -0,0 +1,44 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+// Partially rewritten from https://github.com/Nerei/kinfu_remake
+// Copyright(c) 2012, Anatoly Baksheev. All rights reserved.
```

Comment:
Is it actual for the code below?

Reply:
I think now our code is not based on that old code, everything was rewritten a lot of times.
But `integrate()` & `raycast()` still use some ideas from there (coordinate transform acceleration in integration, surface interpolation in raycasting) , I think it'd be honest to leave this copyright.